examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "MXJob"
+metadata:
+  name: "auto-tuning-job"
+spec:
+  jobMode: MXTune
+  mxReplicaSpecs:
+    TunerTracker:
+      replicas: 1
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+          - name: mxnet
+            image: kingonthestar/auto-tuning:2.0
+            command: ["python3"]
+            args: ["/home/start-job.py"]
+    TunerServer:
+      label: gpu1
```

Comment:
Why do we need such a label?

Reply:
We need use the type of GPU card as label here. Furthermore, we have to update document to guide user how to use the feature. For example, what the label is used for, how user programming their tuning code, how to configure scheduling policy for the feature and so on.